crates/bevy_time/src/timer.rs

```diff
@@ -343,6 +343,21 @@ impl Timer {
         1.0 - self.percent()
     }
 
+    /// Returns the remainig time in seconds
+    ///
+    /// # Examples
+    /// ```
+    /// # use bevy_time::*;
+    /// use std::time::Duration;
+    /// let mut timer = Timer::from_seconds(2.0, false);
+    /// timer.tick(Duration::from_secs_f32(0.5));
+    /// assert_eq!(timer.remaining(), 1.5);
+    /// ```
+    #[inline]
+    pub fn remaining(&self) -> f32 {
```

Comment:
This should return a `Duration` like `elapased` :) Users can easily convert that into an `f32` on their own.